src/controller/README.md

```diff
@@ -0,0 +1,68 @@
+# CHIP controller
+
+There are currently 4 implementations of the CHIP device controller with varying
+support for certain CHIP features.
+
+## Implementations
+
+### iOS
+
+The iOS chip-tool is located in [../darwin/CHIPTool](../darwin/CHIPTool).
+
+### Android
+
+The Android chip-tool is located in [../android/CHIPTool](../android/CHIPTool).
+
+### POSIX CLI
+
+The POSIX CLI chip-tool is located in
+[../../examples/chip-tool](../../examples/chip-tool).
+
+### Python
+
+The Python chip-device-ctrl is located in
+[../controller/python/](../controller/python).
+
+## Feature Overview
+
+| Onboarding | iOS | Android | CLI | Python |
+| ---------- | --- | ------- | --- | ------ |
+| Setup code | y?  | y?      | y   | y      |
```

Comment:
AFAIK it's not possible to use Setup code in the Android app.

Reply:
That is true as of latest master.
Thanks @Damian-Nordic.

Reply:
Changed to n